aces/Loris
# [instrument_list/battery] Clean-up battery retrieval function

## Brief summary of changes

1. Removed getBatteryInstrumentList function from the instrument_list module and made it query the data from the correct place which is the NDB_BVL_Battery class
2. Modified query in NDB_BVL_Battery to remove duplicate returns and avoid "jumpy" test_names depending on if an instrument_order is defined or not (see individual comments below for more details)
3. Removed unused/inaccurate parameters for Battery functions in Battery class. The parameters STAGE, SUBPROJECTID and VISIT LABEL should all be queried from the session itself (Which was already mandatory to define. see `_assertBatterySelected`) but are also either completely unsued in these functions or not necessary at all.

#### Testing instructions (if applicable)
@jesscall @sruthymathew123 @zaliqarosli each of you should test this on at least a handful of timepoints since you all use different functionalities affecting the instrument list module
- QPN uses instruments at the "Not Started" stage which might be affected by these modifications
- IBIS has several test_batteries which diverge from the "defined batteries" which is what the now deleted function was added for in the first place (see #2006). Also can test compatibility with survey instruments
- CCNA uses instrument ordering as well as manual addition of individual instruments to sessions

make sure:
1. instrument ordering works when order is defined in the battery table
2. if ordering is not defined for some or all of the instruments, order should default to Full Name displayed
3. ALL instruments in the flag table for that session appear in the instrument_list module, even instruments which are not in the test_battery defined for that session (ie survey instruments and manually added instruments)
4. Control panel properly checks that all instruments are complete in order to allow pass/fail changes and completion of the session (this functionality depends on being able to query all instruments of the session)
